charts/mozilla-common-voice/templates/deployment.yaml

```diff
@@ -3,6 +3,10 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ .Values.voice_web.deployment_name }}
+  annotations:
```

Comment:
Maybe think about wrapping `annotations:` under the if statement here

Reply:
thanks, I fixed this